modules/oxide/src/less/theme/components/form/autocomplete.less

```diff
@@ -9,11 +9,7 @@
   }
 
   .tox-autocompleter .tox-menu {
-    @menu-border-color: @border-color;
-    @menu-box-shadow: none;
-
-    border-color: @menu-border-color;
-    box-shadow: @menu-box-shadow;
+    box-sizing: border-box;
```

Comment:
This was added because without this when the menu is positioned the size changes because it was previously constrained within the `.tox-autocompleter` wrapper. We also want to ensure the `max-width` is actually `25em` including the padding to keep it consistent with how this worked previously.